templates/aws/pro.html

```diff
@@ -0,0 +1,275 @@
+            <p class="p-matrix__desc">For companies looking to leverage industry benchmarks for hardening, Ubuntu Pro makes two leading profiles available out of the box.</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">AWS-integrated</h3>
+            <p class="p-matrix__desc">Starting in Q1 2020, Ubuntu Pro will include integrations with AWS services including Security Hub, CloudWatch and CloudTrail.</p>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Landscape included</h3>
+            <div class="p-matrix__desc">
+              <p>Landscape provides a complete solution for the patching, compliance reporting and lifecycle management of an Ubuntu cloud fleet via a web UI or API.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">10-year lifetime</h3>
+            <div class="p-matrix__desc">
+              <p>Canonical backs Ubuntu Pro for 10 years<a href="#index-1">*</a>, ensuring security updates are available throughout, with a guaranteed upgrade path.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Priced for the cloud</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro pricing tracks the underlying EC2 cost, starting at 20% for the smallest t3.nano instances, and ramping down to less than 1%.</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+      <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+      <p><a href="/security">Learn more about security for Ubuntu&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2>Launch Ubuntu Pro for AWS</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 18.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 16.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
```

Comment:
The copy doc has "The most commonly deployed Ubuntu LTS on the cloud today, Ubuntu 16.04 LTS released in 2016 with the 4.4 kernel, and is covered with Ubuntu Pro through 2024." for this section.